libraries/bezier-rs/src/subpath/solvers.rs

```diff
@@ -52,7 +54,32 @@ impl Subpath {
 		self.get_segment(segment_index).unwrap().tangent(TValue::Parametric(t))
 	}
 
-	/// Returns a normalized unit vector representing the direction of the normal on the subpath based on the parametric `t`-value provided.
+	/// Returns a list of `t` values that correspond to the self intersection points of the subpath. For each intersection point, the returned `t` value is the smaller of the two that correspond to the point.
+	/// - `error` - For intersections with non-linear beziers, `error` defines the threshold for bounding boxes to be considered an intersection point.
+	/// - `minimum_seperation`: the minimum difference two adjacent `t`-values must have when comparing adjacent `t`-values in sorted order.
+	/// If the comparison condition is not satisfied, the function takes the larger `t`-value of the two
+	///
+	/// **NOTE**: if an intersection were to occur within an `error` distance away from an anchor point, the algorithm will filter that intersection out.
+	// TODO: add iframe to rustdocs
```

Comment:
You can add the iframe as part of this review, just follow the format of the others